dlc-manager/tests/channel_execution_tests.rs

```diff
@@ -362,10 +363,13 @@ fn channel_execution_test(test_params: TestParams, path: TestPath) {
             bob_oracles,
             Arc::clone(&mock_time),
             Arc::clone(&electrs),
+            ManagerOptions::default(),
         )
         .unwrap(),
     ));
 
+    let cet_nsequence: u32 = ManagerOptions::default().cet_nsequence;
```

Comment:
I'm not sure what you mean here. I see that cet_nsequence used to be a CONST, and now because I'm grabbing this off the struct I can only store it as a let. Can you give me an example? I'm not sure what you mean by using the DEFAULT prefix. Thanks

Reply:
What I meant was that you could keep the const and use them to initialize the default values of the struct, and make them accessible so that someone using the default values doesn't have to initialize a struct for nothing just to access the default values.

Reply:
Ah, I understand now. Yes, updated